src/hooks/useContract.ts

```diff
@@ -68,3 +71,13 @@ export function useRewardViaMultiplierContract(address?: string, withSignerIfPos
 export function usePairContract(pairAddress?: string, withSignerIfPossible?: boolean): Contract | null {
   return useContract(pairAddress, IPangolinPair.abi, withSignerIfPossible);
 }
+
+export function usePngContract(): Contract | null {
+  const chainId = useChainId();
+  return useContract(chainId ? PNG[chainId].address : undefined, Png.abi, true);
```

Comment:
looks like `useChainId` always returns chainId. we might not need `undefined` condition here